frontend/src/features/properties/components/forms/subforms/PidPinForm.tsx

```diff
@@ -44,7 +57,12 @@ const PidPinForm: FunctionComponent<PidPinProps> = (props: PidPinProps) => {
           tooltip={PidTooltip}
           disabled={props.disabled}
           pattern={RegExp(/^[\d\- ]*$/)}
-          onBlurFormatter={(pid: string) => pid.replace(pid, pidFormatter(pid))}
+          onBlurFormatter={(pid: string) => {
+            if (pid && getIn(touched, withNameSpace('pid'))) {
+              debouncedHandlePidChange(pid);
```

Comment:
Does this remove the `-` dashes when making a request to the layer?

Reply:
It does at the container level via .replace(/-/g, '')